informatici/openhospital-api
# OP-1309 | Create endpoints for CRUD on GroupPermissions

See OP-1309

### Updates
- update endpoint `POST /users/groups` to accept a DTO containing the list of `PermissionDTO` to assign to user group being created.
- update endpoint `PUT/users/groups` to accept a DTO containing the updated list of `PermissionDTO` to assign to user group being updated.

### New
- e2e tests for `UserController`
- `POST /users/groups/{group_code}/permissions/{id}` to assign a permission to a user group
- `DELETE /users/groups/{group_code}/permissions/{id}` to revoke a permission from a user group

### How to test
Use the PR [1391](https://github.com/informatici/openhospital-core/pull/1391) on the CORE.
1. Run the app
2. Check if tests are passing
3. Use swagger to manually test the two updated endpoints
